extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -116,6 +116,11 @@ amp-carousel .amp-carousel-button.amp-disabled {
   width: 100% !important;
 }
 
+.-amp-slide-item > * {
+  height: 100%;
+  width: 100%;
+}
+
```

Comment:
@dvoytenko , this works perfectly fine , and the element can override the width and height based on their layouts!